pkg/util/helpers.go

```diff
@@ -23,12 +23,12 @@ import (
 )
 
 // GenerateConditionChangeEvent generates an event for condition change.
-func GenerateConditionChangeEvent(t string, status types.ConditionStatus, reason string, timestamp time.Time) types.Event {
+func GenerateConditionChangeEvent(t string, status types.ConditionStatus, reason, message string, timestamp time.Time) types.Event {
 	return types.Event{
 		Severity:  types.Info,
 		Timestamp: timestamp,
 		Reason:    reason,
-		Message:   fmt.Sprintf("Node condition %s is now: %s, reason: %s", t, status, reason),
+		Message:   fmt.Sprintf("Node condition %s is now: %s, reason: %s, message: %s", t, status, reason, message),
```

Comment:
As mentioned in the PR summary, I'm totally open to changing the wording and formatting here, just let me know. I would also like to know if this message should be truncated.

Reply:
I suggest using `%q` instead of `%s`, since the message might include special render characters (such as \n) and displaying the value makes sense.